app/assets/javascripts/components/timeline/timeline.jsx

```diff
@@ -60,6 +64,10 @@ const Timeline = createReactClass({
     return week.blocks;
   },
 
+  usingCustomTitles() {
+    return this.props.weeks.find(week => week.title) !== undefined;
```

Comment:
So, this line is trying to determine whether or not _anything_ has a title, correct? If that's the case, the following might provide a more clear intention:
```js
return this.props.weeks.some(week => week.title)
```
Alternatively, you could just return a boolean value with your `.find()` result instead of having to compare to `undefined`:
```js
return !!this.props.weeks.find(week => week.title)
```